src/renderer/l10n/de.js

```diff
@@ -16,18 +16,21 @@ export default {
   addressBook: 'Adressbuch',
   all: 'Alle',
   amount: 'Menge',
+  amountCommitted: 'Amount Committed',
```

Comment:
If you hadn't added these in the other language files, it would have defaulted back to English, but this makes it easier to see the ones in english still for the language. When looking at the latest version of this file. We probably should have a utility though that sees the ones missing and creates a file with the ones missing to give to translators, and also some utility to merge the newly translated strings back in. For now this is fine though.